fava/templates/source.html

```diff
@@ -15,7 +15,7 @@
                 </tr>
             </thead>
             <tbody>
-                {% for source_file in api.source_files() %}
+                {% for source_file in api.source_files()|sort %}
```

Comment:
@corani `api.source_files()` returns the source files in the order that beancount loads them - so your main file first. Are you sure you want them to be sorted by name instead?